workspaces/wheel-of-names/plugins/wheel-of-names/README.md

```diff
@@ -20,6 +20,26 @@ yarn --cwd packages/app add @backstage-community/plugin-wheel-of-names
 
 ## Configuration
 
+### New Frontend System (Recommended)
+
+If you are using Backstage's [new frontend system](https://backstage.io/docs/frontend-system/), add the plugin to your `packages/app/src/App.tsx`:
+
+```tsx
+import { createApp } from '@backstage/app-defaults';
+import { wheelOfNamesPlugin } from '@backstage-community/plugin-wheel-of-names/alpha';
+
+const app = createApp({
+  features: [
+    // ... other features
+    wheelOfNamesPlugin,
+  ],
+});
+```
```

Comment:
I think auto-discovery of plugins is now enabled by default so perhaps we could remove this entire block? 🤔